tests/test_operator_complex.cpp

```diff
@@ -196,9 +244,10 @@ int main() {
     test_passes &= test_valid_types<test_add_assign>(Q, INFINITY, INFINITY,
                                                      INFINITY, INFINITY);
 
-    test_passes &= test_valid_types<test_add_assign>(Q, NAN, 2.02, NAN, 2.02);
-    test_passes &= test_valid_types<test_add_assign>(Q, 4.42, NAN, 4.42, NAN);
-    test_passes &= test_valid_types<test_add_assign>(Q, NAN, NAN, NAN, NAN);
+    test_passes &= test_valid_types<test_add_assign>(Q, NAN, 2.02,
+    NAN, 2.02); test_passes &= test_valid_types<test_add_assign>(Q, 4.42,
+    NAN, 4.42, NAN); test_passes &= test_valid_types<test_add_assign>(Q, NAN,
+    NAN, NAN, NAN);
```

Comment:
```suggestion
    test_passes &= test_valid_types<test_add_assign>(Q, NAN, 2.02,
    NAN, 2.02); 
    test_passes &= test_valid_types<test_add_assign>(Q, 4.42,
    NAN, 4.42, NAN); 
    test_passes &= test_valid_types<test_add_assign>(Q, NAN,
    NAN, NAN, NAN);
```

And same for the 3 other bellows

Reply:
Sorry, I should have caught this. This has been fixed with clang-format.